numpy/distutils/command/config.py

```diff
@@ -449,8 +454,24 @@ def get_output(self, body, headers=None, include_dirs=None,
                 grabber.restore()
                 raise
             exe = os.path.join('.', exe)
-            exitstatus, output = exec_command(exe, execute_in='.',
-                                              use_tee=use_tee)
+            try:
+                # specify cwd arg for consistency with
+                # historic usage pattern of exec_command()
+                # also, note that exe appears to be a string,
+                # which exec_command() handled, but we now
+                # use a list for check_output() -- this assumes
+                # that exe is always a single command
+                output = subprocess.check_output([exe], cwd='.')
```

Comment:
Maybe I could still do better here -- `exe.split(' ')` actually produces a list even if the string has no whitespace (single command), so is perhaps arguably more robust here?

Reply:
This enclosing function has been deprecate for 10 years - can we just remove it?

Reply:
`git grep 'get_output'`

```
numpy/distutils/command/config.py:    def get_output(self, body, headers=None, include_dirs=None,
numpy/distutils/system_info.py:        s, o = c.get_output(atlas_version_c_text,
numpy/distutils/system_info.py:            s, o = c.get_output(atlas_version_c_text,
```

Reply:
That said, I just tried deleting `get_output` locally and everything seems fine test-wise, but presumably we'd want to get rid of those other two calls--would we then get rid of those other enclosing functions or try to replace with something else?

Reply:
I think it best to leave it in for this PR and leave removal to another. Things are already complicated enough, and if we need to revert some bit it is easier if there are separate PRs.